renpy/python.py

```diff
@@ -353,7 +353,19 @@ def reset_store_changes(name):
 # the rollback list before rollback or serialization.
 
 
-class NoRollback(object):
+class SlottedNoRollback(object):
+    """
+    :doc: norollback class
```

Comment:
There should not be two identical entries in the seme place in the doc for different classes. Either the new one is explained in a way highlighting its specificities, or it doesn't get documented.

Reply:
Would adding the following sentence be appropiate?
> SlottedNoRollback is designed to be used by classes that want to benefit from the reduced memory footprint that a class with `__slots__` offers.